tsconfig.json

```diff
@@ -15,6 +22,6 @@
     "jsx": "preserve",
     "incremental": true
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "spaghetti.js", "src/common/components/Layout/Header/Header.jsx"],
```

Comment:
I don't see there is any file existing in this path `"src/common/components/Layout/Header/Header.jsx"`